### PR TITLE
PC-1423: Q3 Scottish Data Dump

### PR DESCRIPTION
**Pull request by Glenn-Clarke**
[Link to Jira ticket](https://beisdigital.atlassian.net/browse/PC-1423)

# Description

Update the month names to the new quarter: 1st October -> 1st February

# Checklist

- [x] I have made any necessary updates to the documentation
- [x] I have checked there are no unnecessary IDE warnings in this PR
- [x] I have checked there are no unintentional line ending changes
- [x] I have added tests where applicable
- [x] If I have made any changes to the code, I have used the IDE auto-formatter on it
- [x] If I have made any changes to Python files (even if not changing any content strings), I have run `make extract-translations`
- [x] If I have made any changes to website flow, I have updated the [Flow Miro Board](https://miro.com/app/board/uXjVK4AgQ3E=/)
- [x] If I have made any changes to website flow, I have checked forward and back behaviour is still consistent

# Screenshots
{FE99CF57-DA4E-418D-9B30-8CBCB9AEAE5E}

### CODE REVIEW

File: help_to_heat/utils.py [hunk 1]
```diff
@@ -240,11 +240,12 @@ def get_current_and_next_month_names(month_names):
...
+# Q1 january -> may
+# Q2 april -> august
+# Q3 july -> november
+# Q4 october -> february
 # we update the month names manually on each scottish data dump
 # this is to tell the user the month of the last dump, and expected month of next dump
+# the quarters overlap by one month, as the dump takes place roughly at the end of a month
 def get_current_and_next_quarter_month_names(month_names):
-    return month_names[6], month_names[9]
+    return month_names[9], month_names[1]
```
Comment (jdgage) [inline help_to_heat/utils.py:251]: I think this language is confusing now - Q1 is not January-May for example. Worth reviewing: what are we actually trying to say with these?

Are we definitely right to say October?

Comment (Glenn-Clarke) [reply help_to_heat/utils.py:251]: Hmm, you're right. I'll have a rethink of the wording.

I think what I'm trying to get across with this comment is which pairs of months to use in which quarter, ultimately. But yeah there's a disconnect between the quarter we're *in*, and the quarter of the data dump you're performing.

Comment (Glenn-Clarke) [reply help_to_heat/utils.py]: I've attempted to be clearer about the purpose of the comment, and how it should be used. I've also renamed the function to be a bit closer to what it actually is doing.

File: help_to_heat/frontdoor/views.py
[the remark this answers is not in the document]
Comment (Glenn-Clarke) [reply help_to_heat/frontdoor/views.py:1]: I've updated the variable names through to the translations & html files.

**Review by jdgage — APPROVED** [review]

**Pull request merged**